nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -322,12 +322,21 @@ in {
           Please migrate your configuration to config.services.nextcloud.poolSettings.
         '')
         ++ (optional (versionOlder cfg.package.version "18") ''
+          A legacy Nextcloud install (from before NixOS 20.03) may be installed.
```

Comment:
I don't like the wording "legacy Nextcloud install" as it kinda says "legacy nextcloud", but it's "our old kind of setup". Any proposals?

Reply:
`An older version of Nextcloud`?

Reply:
What I'm thinking here is that the fault lies in **our setup** of Nextcloud, not in Nextcloud being of an older version.